src/Data/Quantity/Math.purs

```diff
@@ -59,12 +71,21 @@ lift2 f q1 q2 = do
 acos ∷ Quantity → Result
 acos = lift Decimal.acos
 
+asec ∷ Quantity → Result
+asec = lift \x -> Decimal.acos (one / x)
+
 asin ∷ Quantity → Result
 asin = lift Decimal.asin
 
+acsc ∷ Quantity → Result
+acsc = lift \x -> Decimal.asin (one / x)
+
 atan ∷ Quantity → Result
 atan = lift Decimal.atan
 
+acot ∷ Quantity → Result
+acot = lift \x -> Decimal.atan (one / x)
```

Comment:
`acot` is a multi-valued function. The definition that you give here is one possible option to define `acot`. It has a range (-pi/2, pi/2] and a discontinuity at x=0.

There is another convention which has a range (0, pi) which has the advantage that it is continuous along the real line. Since Insect has no support for complex numbers, I wonder if we should use the latter convention, which is:

![image](https://user-images.githubusercontent.com/4209276/160300368-bba9e987-23f7-4edf-b932-2046ee569778.png)
(only found a German version)

Source: https://mathworld.wolfram.com/InverseCotangent.html

Reply:
Done. That definition was lifted from Qalculate (as stated in my other comment), which does support complex numbers.